src/preprocess.jl

```diff
@@ -46,6 +46,17 @@ function add_node_to_node_set!(mesh::Mesh, set_name, nids...)
         mesh.node_sets[set_name] = Set{Int64}()
     end
     push!(mesh.node_sets[set_name], nids...)
+    return
+end
+
+""" Create a new node set from nodes in element set. """
+function create_node_set_from_element_set!(mesh::Mesh, set_name)
+    node_ids = Set{Int64}()
+    for elid in mesh.element_sets[set_name]
```

Comment:
You are right. In practice Int == Int64, but "Julia also defines the types Int and UInt, which are aliases for the system’s signed and unsigned native integer types respectively", so I think it would be better to use Int, Float instead of Int64 and Float64.